src/ActorUtil.h

```diff
@@ -42,6 +42,19 @@ enum FileType
 	NUM_FileType,
 	FileType_Invalid
 };
+namespace std
+{
+	template<>
+		struct hash<FileType>
+	{
+		typedef FileType argument_type;
+		typedef std::size_t result_type;
+		result_type operator()(argument_type const& s) const
```

Comment:
I'm not seeing the point of this typedef. Granted, I think you use this in a lot of places. What is the point?